Dfe.PrepareConversions/Dfe.PrepareConversions.Data.Tests/Models/AcademisationApplication/PopulateSchoolDetailsTests.cs

```diff
@@ -0,0 +1,69 @@
+﻿using ApplyToBecome.Data.Models.AcademisationApplication;
+using Dfe.PrepareConversions.Data.Models.Application;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Xunit;
+using static ApplyToBecome.Data.Models.AcademisationApplication.AcademisationApplication;
+
+namespace Dfe.PrepareConversions.Data.Tests.Models.AcademisationApplication
+{
+   public class PopulateSchoolDetailsTests
+   {
+      [Fact]
+      public void PopulateSchoolDetails_Should_Populate_Fields_Correctly()
+      {
+         // Arrange
+         var academisationApplicationSchool = new School
+         {
+            SchoolName = "Old School Name",
+            SchoolConversionContactHeadName = "Head Name",
+            SchoolConversionContactHeadEmail = "head@email.com",
+            SchoolConversionContactHeadTel = "1234567890",
+            SchoolConversionContactRole = "Role",
+            SchoolConversionContactChairName = "Chair Name",
+            SchoolConversionContactChairEmail = "chair@email.com",
+            SchoolConversionContactChairTel = "0987654321",
+            SchoolConversionMainContactOtherName = "Other Name",
+            SchoolConversionMainContactOtherEmail = "other@email.com",
+            SchoolConversionMainContactOtherTelephone = "1111111111",
+            SchoolConversionApproverContactName = "Approver Name",
+            SchoolConversionApproverContactEmail = "approver@email.com",
+            SchoolConversionTargetDateSpecified = true,
+            SchoolConversionTargetDate = new DateTime(2023, 1, 1),
+            SchoolConversionTargetDateExplained = "Target Date Explained",
+            ApplicationJoinTrustReason = "Reasons",
+            ConversionChangeNamePlanned = true,
+            ProposedNewSchoolName = "New School Name"
+         };
+
+         var academiesApplicationSchool = new ApplyingSchool();
+
+         // Act
+         PopulateSchoolDetails(academiesApplicationSchool, academisationApplicationSchool);
+
+         // Assert
+         Assert.Equal("Old School Name", academiesApplicationSchool.SchoolName);
```

Comment:
Shouldly has the option to use - BeEquivalentTo. But this might not help for this particular case if the objects to be compared are quite different.